### DISCUSSION TITLE
How it works?

### DISCUSSION THREAD

**Discussion by glyh** [afnanenayet/diffsitter#329] [Q&A] [answered]
As title, thanks a lot!

**Comment by sleexyz** [comment, marked as answer]
In the docs of a related tool (difftastic) it says that diffsitter uses "LCS diffing on the leaves of the syntax tree"

https://difftastic.wilfred.me.uk/tree_diffing.html#diffsitter-2020

**Reply by afnanenayet** [reply]
Ack sorry I totally forgot to respond to this. Thanks for answering @sleexyz!
